cattle/plugins/docker/compute.py

```diff
@@ -215,6 +215,17 @@ def _determine_state(self, container):
             # Unknown. Assume running and state should sync up eventually.
             return 'running'
 
+    def _get_host_labels(self):
+        try:
+            labels = self.host_info.host_labels()
+        except:
+            log.exception("Error getting host labels")
```

Comment:
I'm following along with what I did previously to the host stats. I don't know that I have ever seen the error =D would you suggest something else? Given the number of failures that can occur below, we definitely need the try/catch. 

I'm open for a different method, I know polluted logs aren't helpful.